api/handlers/containerhandler.py

```diff
@@ -251,6 +255,13 @@ def put(self, cont_name, **kwargs):
 
         payload = self.request.json_body
         payload_validator(payload, 'PUT')
+        # Check if any payload keys are a propogated property, ensure they all are
+        rec = False
+        if set(payload.keys()).intersection(set(self.config['propagated_properties'])):
```

Comment:
This will fail if the `self.config` doesn't have a 'propagated_properties`(and this is true for acquisitions for example).
Doing something like `self.config.get('propageted_properties', [])` will fix it.